include/ccf/http_configuration.h

```diff
@@ -43,4 +43,9 @@ namespace http
     max_concurrent_streams_count,
     initial_window_size,
     max_frame_size);
+
+  static ParserConfiguration permissive_configuration()
+  {
+    return ParserConfiguration{{"1GB"}, {"100MB"}, 1024, 1, {"64KB"}, {"16MB"}};
```

Comment:
Where do these numbers come from? What if a node sets its limits even higher than this?

Reply:
They're what I think we can tolerate at best with the current enclave size without crashing the primary. I think we can re-evaluate those if we raise that, but I don't think it makes sense to raise the limits up to the point where we're confident we're going to get a crash.